views/todotask.html

```diff
@@ -214,8 +268,92 @@ <h1>Your Task</h1>
 			});
 
 
+            // show your task
+
+             $('.TaskBtn').click(function(e) {
+				e.preventDefault();
+
+				$('.taskListWrap').toggle();
+
+				var taskData = localStorage.getItem("user");
+				var taskData = JSON.parse(taskData);
+				var userid = taskData.userid;
+
+				var request = $.ajax({
+					"url": "http://localhost:3001/viewTask",
+					"type": "post",
+					"data": {
+						"userid" :userid
+					}
+				});
+
+				request.success(function(data){
+					console.log(data);
+
+					$('.taskList').empty();
+					for (var i = 0; i < data.length; i++) {
+						$('.taskList').append('<tr>\
+							<td class="text-center">'+data[i]["task"]+'</td>\
+							<td class="text-center">'+data[i]["date"]+'</td>\
+							</tr>');
+					}
+					
+				});
+
+				request.fail(function(error){
+					console.log("error");
+				});
+			});
+
 
 		});
+             $(document).ready(function(){
+			var userData = localStorage.getItem('user');
+			var userData = JSON.parse(userData);
+			console.log(userData);
+			$('.nameField').empty();
+			$('.nameField').html(userData.name);
+		      });
+			$('.logOutBtn').click(function(e){
+				 e.preventDefault();
+ 				//console.log("dkjdkjdkjd");
+ 				localStorage.removeItem("user");
+				 window.location.replace('index.html');
+				});
+			/* For preview image */  
+			$(document).ready(function(){
+            $('.previewImage').change(function(event){
+
+					var userData = JSON.parse(localStorage.getItem('user'));
+					var user_id = userData.user_id;
+
+					var file = event.target.files[0];
+					$('.previewImageContainer').empty();
+
+					var formData = new FormData();
+					formData.append('image', file);
+					formData.append('user_id', user_id);
+
+					$.ajax({
+					type: 'POST',
+					url: 'http://localhost:3001/uploadImage',
+					dataType: "json",
+					data: formData,
+					async: false,
+					processData: false,
+					contentType: false,
+					success: function (data) {
+						console.log(data);
+					}
+				});
+				$('.previewImageContainer').html('<img src='+URL.createObjectURL(file)+' width="150px" height="150px">');
+			});
+        });
+
+		
+	
+			
+
 
 		// $(window).load(function(){
```

Comment:
remove the commented code if not required

Reply:
done..